zc_common/remote_resource/mixins.py

```diff
@@ -0,0 +1,23 @@
+"""
+Class Mixins.
+"""
+from django.db import IntegrityError
+from django.http import Http404
+
+
+class MultipleIDMixin(object):
+    """
+    Override get_queryset for multiple id support
+    """
+    def get_queryset(self):
+        """
+        Override :meth:``get_queryset``
+        """
+        if hasattr(self.request, 'query_params'):
+            ids = dict(self.request.query_params).getlist('ids[]')
+            if ids:
+                try:
+                    self.queryset = self.queryset.filter(id__in=ids)
```

Comment:
we should be using `pk__in` in case the model doesn't have the standard primary key field.